services/app-graphql/src/schema.graphql

```diff
@@ -462,6 +462,16 @@ type HelpdeskUser {
     familyName: String!
 }
 
+"BusinessOwnerUser is a user that ..."
```

Comment:
Halp! What exactly do we want to be the description here?

Reply:
The Business Owner is the root of our role hierarchy with the power to approve new admin accounts